types/node/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for non-npm package Node.js 13.1
+// Type definitions for non-npm package Node.js 13.5
```

Comment:
based on PR title this should be 13.6 now

Reply:
Welp, that was merged too quickly 😛.
Will fix with 13.7 PR :)